Dockerfile

```diff
@@ -31,10 +31,17 @@ COPY 0001-wic-Adjust-cmd-line-format-to-debugfs-1.45.6.patch /root/oe
 RUN cd /root/oe \
   && patch scripts/lib/wic/engine.py 0001-wic-Adjust-cmd-line-format-to-debugfs-1.45.6.patch
 
-RUN apk add python3 py3-pip libffi-dev make openssl-dev gcc libc-dev python3-dev
+RUN apk add python3 py3-pip libffi-dev make openssl-dev gcc libc-dev python3-dev rust cargo
 RUN pip3 install docker-compose==1.27 expandvars==0.6.5
 
-## Stage 2
+
+FROM golang:alpine
+RUN apk add gcc git glib-dev make musl-dev
+RUN git clone https://github.com/foundriesio/ostreehub /ostreehub && \
+	cd /ostreehub && git checkout 973d7b8eae07bfc0f93698b9234922e372b87e01
```

Comment:
I think you need to update the commit hash or just specify HEAD to get fiopush version that supports the summary update.

Reply:
i force pushed with the new sha.